crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -101,5 +111,19 @@ def failed?
     def upgrade_script_path
       Rails.root.join("..", "bin", "upgrade_admin_server.sh")
     end
+
+    def addon_installed?(addon)
+      case addon
+      when "ceph"
+        CephService
+      when "ha"
+        PacemakerService
+      else
+        return false
+      end
+      true
```

Comment:
ah, so you try to resolve `CephService`, if is defined you ignore the return value and return true, if is not defined a NameError happends and return false.